src/main/java/com/splunk/logging/HttpEventCollectorLog4jAppender.java

```diff
@@ -38,6 +38,7 @@
 public final class HttpEventCollectorLog4jAppender extends AbstractAppender
 {
     private HttpEventCollectorSender sender = null;
+    private boolean sendFullThrowable = false;
```

Comment:
I don't think the verbosity is helpful here, let's rename all members to `sendThrowable` or `send_throwable`